### PR TITLE
Fixed issues with ipmessaging

### PR DESCRIPTION
**Pull request by senthgit**
- update calls
- Tests are also updated

### CODE REVIEW

File: twilio/rest/resources/ip_messaging/roles.py [hunk 1]
```diff
@@ -35,3 +41,27 @@ def delete(self, sid):
...
+    def create(self, friendly_name, type, permission):
+        """
+        Creates a Role
+        :param str friendly_name: Human readable name to the Role
+        :param str type: Type of role - deployment or channel
+        :param str permission: Set of permissions for the role
+        """
+        kwargs = {
+            "friendly_name": friendly_name,
+            "type": type,
```
Comment (jingming) [inline twilio/rest/resources/ip_messaging/roles.py]: does this override the `type` method?

Comment (senthgit) [inline twilio/rest/resources/ip_messaging/roles.py]: I dont think so, but is not the right parameter name. I will change it.

File: twilio/rest/__init__.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
 from .base import set_twilio_proxy
 from .client import TwilioRestClient
+from .conversations import TwilioConversationsClient
```
Comment (jingming) [inline twilio/rest/__init__.py]: this required?

Comment (senthgit) [inline twilio/rest/__init__.py]: yes this makes importing TwilioConversationsClient, TwilioIpMessagingClient.. simpler. 
I will add these two to pyflakes list.

**Comment by jingming** [comment]
+1

**Pull request merged**